k2/csrc/tensor.h

```diff
@@ -147,26 +148,35 @@ class Tensor {
         reinterpret_cast<char *>(impl_->data->data) + impl_->bytes_offset);
   }
 
+  void *Data() {
+    return reinterpret_cast<char *>(impl_->data->data) + impl_->bytes_offset;
+  }
+
+  const void *Data() const {
```

Comment:
I think we can just make the other version const, and return non-const void.
It has turned out to be unworkable to let 'const' reflect the const-ness of the underlying data.
If anything it will just mean that the metadata is not changed.

Reply:
Fixed.